packages/pn-personafisica-webapp/src/component/Notifications/NotificationPayment.tsx

```diff
@@ -319,7 +319,7 @@ const NotificationPayment: React.FC<Props> = ({ iun, notificationPayment, onDocu
   const attachments = getAttachmentsData();
 
   return (
-    <Paper sx={{ padding: '24px', marginBottom: '1rem' }} className="paperContainer">
+    <Paper sx={{ px: 3, pt: 3, marginBottom: '1rem' }} className="paperContainer">
```

Comment:
pt deve diventare py, altrimenti negli altri casi in cui il pagamento va in errore, il messaggio di errore è attaccato al bordo inferiore della card